rcl/CMakeLists.txt

```diff
@@ -12,9 +12,12 @@ find_package(rosidl_generator_c REQUIRED)
 
 include_directories(include)
 
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Fixed in https://github.com/ros2/rcl/pull/141/commits/4e092bc560ff7eeaa242a6da05ae13ae6cfcb3e8